docs/modules/ROOT/pages/sentinel.adoc

```diff
@@ -0,0 +1,211 @@
+[[sentinel]]
+= Sentinel
+
+_The Defender Sentinel service allows you to monitor transactions for Addresses by specfying Events, Functions, and Transaction conditions and notifying via email, slack, and autotasks (more to come)._
+
+[[use-cases]]
+== Use cases
+
+Use a Sentinel whenever you need to know about or respond to transactions involving your smart contract.  Your Sentinel will watch every transaction send the transactions you care about to your notification methods of choice.
+
+* *Monitor* your *sensitive functions* like transferOwnership, pause
+* *Alert* when transactions are failing against your smart contract
+* *Respond* to transactions by executing *Autotask* logic when key events happen 
+* *Integrate* with your *existing tools* through slack, email or custom *Autotask* integration
+* *Know* when an unexpected volume of transactions occur
+
+[[whats-in-an-sentinel]]
+== What's in an Sentinel?
+
+A Sentinel watches transactions for a given address, filters those transactions by any conditions that you specify, then notifies you when those transactions occur.
+
+IMPORTANT: At this time, Sentinels do not monitor internal function calls.
+
+[[specify-address]]
+=== Specifying an Address
+
+* *Address*: Choose the address of either a Smart Contract or a normal key-pair address to monitor.
+
+* *ABI*: Specify the ABI for your contract.  We will attempt auto-discovery of the ABI if it is verified.
+
+* *Confirmation Blocks*: If you want to be notified only after a certain level of confidence that the transaction is accepted, choose a higher confirmation block level.   If you want to be notified as soon as possible and are tolerant to re-orgs, then choose *Latest* 
+
+[[matching-rules]]
+=== Matching Rules
+
+For a Transaction to trigger a notification, it must satisfy *ALL* of the following:
+
+* Transaction *MUST* have a To, From, or Address (from log) that matches the configured address
+
+* If a *Transaction Condition* is specified
+
+    ** The Transaction *MUST* match the *Transaction Condition*.
+
+* If *Events* are selected
+
+    ** The Transaction *MUST* emit a selected Event and match its *Event Condition* (if any)
+
+* If *Functions* are selected
+
+    ** The Transaction *MUST* invoke a selected Function and match its *Function Condition* (if any)
+
+[[choose-events-and-functions]]
+=== Events and Functions
+
+Select any Events and Function calls that you wish to monitor. Selecting Events and Functions acts as an OR clause, in that if a transaction matches ANY of the selected Events or Functions, it will trigger notification.
+
+NOTE: An ABI is required to configure Events and Function detection
+
+NOTE: If no Events or Functions are specified, then the Sentinel will monitor ALL transactions to or from your monitored address including those that do not involve functions or events (e.g., sending ether to your contract's address).  If Events or Functions are selected, then ONLY those that are selected are monitored.
```

Comment:
```suggestion
NOTE: If no Events or Functions are specified, then the Sentinel will monitor ALL transactions to or from your monitored address.
```
I wouldn't go into that much detail. Also, note that it's possible to have an event emitted when sending ether to the contract address by doing so in the [receive function](https://docs.soliditylang.org/en/v0.6.2/contracts.html#receive-ether-function).